Lesson 5 - CSS/code/exercises/exercise 1/theme.css

```diff
@@ -1,3 +1,118 @@
-.row {
+.row 
+{
     display: block;

```

Comment:
Regarding to `row` and `column` styles - you could try to experiment with `display: flex` https://css-tricks.com/snippets/css/a-guide-to-flexbox/